packages/rpc-graphql/src/schema/root.ts

```diff
@@ -2,6 +2,7 @@ export const rootTypeDefs = /* GraphQL */ `
     type Query {
         account(address: Address!, commitment: Commitment, minContextSlot: Slot): Account
         block(slot: Slot!, commitment: CommitmentWithoutProcessed): Block
+        multipleAccounts(addresses: [Address!]!, commitment: Commitment, minContextSlot: Slot): [Account]
```

Comment:
This field, as written, returns a nullable array. This will force the consumer to treat `null` and `[]` the same with two different checks. What you actually want is to unconditionally return an array, maybe empty, but in any case with `null` for every unfetchable account.
```suggestion
        multipleAccounts(addresses: [Address!]!, commitment: Commitment, minContextSlot: Slot): [Account]!
```